Assets/Scripts/Managers/GameManager.cs

```diff
@@ -43,4 +47,36 @@ public void SaveData(GameData data)
         data.permanentInventory = permanentInventory.Items;
         data.currency = Currency;
     }
+
+    public void PauseGame()
+    {
+        if (pauseMenuCanvas == null) return;
+
+        if (pauseMenuCanvas.enabled)
+        {
+            ResumeGame();
+        }
+        else
+        {
+            Time.timeScale = 0;
+            ShowCursor();
+            pauseMenuCanvas.enabled = true;
+        }
+    }
+
+    public void ResumeGame()
+    {
+        if (pauseMenuCanvas == null) return;
+
+        Time.timeScale = 1;
+        HideCursor();
+        pauseMenuCanvas.enabled = false;
+    }
```

Comment:
It feels like these should be merged into one function that takes a `bool` as to whether or not the game should be paused. Can add a `TogglePause()` which calls `PauseGame(!IsPaused)` to handle other scripts not caring what state the game is in